src/goto-symex/features.cpp

```diff
@@ -0,0 +1,96 @@
+#include "irep2/irep2_type.h"
+#include <goto-symex/features.h>
+
+bool ssa_features::run(symex_target_equationt::SSA_stepst &steps)
+{
+
+  features.clear();
+
+  for (const auto &step : steps)
+  {
+    const expr2tc expressions[2] = {step.cond, step.guard};
+    for (const expr2tc &e : expressions)
```

Comment:
You could use
```c++
for (const expr2tc &e : std::array { step.cond, step.guard })
```
to make it easier to read. If not that's fine, too.

Reply:
Just changed